__tests__/notifications.spec.js

```diff
@@ -0,0 +1,98 @@
+const request = require('supertest');
+const express = require('express');
+const notificationRoutes = require('../routes/notificationRoutes');
+const { User, Event } = require('../database');
```

Comment:
Where is User and Event from? These functions do no exist in database, and database itself is a dir, so you might have meant database/database...